### PR TITLE
UI fixes

### PR DESCRIPTION
**Pull request by Hussain-7**

### CODE REVIEW

File: ui/src/App.js [hunk 1]
```diff
@@ -33,19 +33,31 @@ function App() {
 
   const { activeCard, openExpandModal, addFormLoader, checkIcon } = state;
   useEffect(() => {
+    console.log('Inside Use effect:', activeTab);
```
Comment (danishyasin33) [inline ui/src/App.js:36]: Remove log

File: contract/src/secondaryStoreWrapper.js [hunk 2]
```diff
@@ -229,13 +222,9 @@ const start = (zcf) => {
...
     getAvailableOffers,
     makeMatchingInvitation,
-  });
-  // CMT (haseeb.asim@robor.systems): Creator facet for the contract. It provides functions that can only be used by the creator of the contract.
-  const creatorFacet = Far('CreatorFacetForSwapInvitation', {
     updateAvailableOffers,
   });
-
-  return harden({ publicFacet, creatorFacet });
+  return harden({ publicFacet });
```
Comment (danishyasin33) [inline contract/src/secondaryStoreWrapper.js:239]: Why are we removing creatorFacet?

File: ui/src/components/CardDisplay.jsx [hunk 3]
```diff
@@ -53,19 +58,38 @@ const CardDisplay = ({ handleClick, handleNFTMint }) => {
     const ids = params?.userCards?.map((card) => card.id);
     // change !== to === to filter user owned cards from secondaryMarketplace
     const arr = params?.userOffers?.filter(
-      (card) => ids.indexOf(card.id) !== -1,
+      (card) => ids.indexOf(card.id) === -1,
```
Comment (danishyasin33) [inline ui/src/components/CardDisplay.jsx:56]: Why?

File: ui/src/components/CardDisplay.jsx [hunk 4]
```diff
@@ -90,12 +114,14 @@ const CardDisplay = ({ handleClick, handleNFTMint }) => {
...
+      menuOptions = ['Name', 'Author'];
+      const filteredList = getFilteredList(secondaryCards, menuOption);
       console.log(userOffers, 'userCards');
       cards =
-        secondaryCards?.length !== 0 ? (
+        filteredList?.length !== 0 ? (
           <div className="grid sm:grid-cols-1  md:grid-cols-2 xl:grid-cols-3  gap-x-6 gap-y-10">
-            {secondaryCards?.map((cardDetail) => {
+            {filteredList?.map((cardDetail) => {
               console.log(cardDetail, 'inside map ');
```
Comment (danishyasin33) [inline ui/src/components/CardDisplay.jsx]: remove log

File: ui/src/components/Header.jsx [hunk 5]
```diff
@@ -8,6 +8,7 @@ const Header = () => {
   const { connected: walletConnected, activeTab } = state;
   const history = useHistory();
   const walletStatus = walletConnected ? 'Connected' : 'Not connected';
+  // dispatch(setActiveTab(0));
```
Comment (danishyasin33) [inline ui/src/components/Header.jsx:11]: delete comment

File: ui/src/components/CardDisplay.jsx (hunk 3 above)
Comment (Hussain-7) [reply ui/src/components/CardDisplay.jsx:56]: So now the user won't able to see nfts he himself puts on sale on the secondary marketplace.

File: ui/src/components/CardDisplay.jsx (hunk 4 above)
Comment (Hussain-7) [reply ui/src/components/CardDisplay.jsx]: Ok

File: contract/src/secondaryStoreWrapper.js (hunk 2 above)
Comment (Hussain-7) [reply contract/src/secondaryStoreWrapper.js:239]: The update available offer is something that should be used by every user.I Don't know why Haseeb had put this in creator faucet.This is what also causing issue.
The agoric team was also not able to remove item from sale due to this.

**Comment by danishyasin33** [comment]
stael

**Comment by danishyasin33** [comment]
stale

**Pull request closed without merging**